packages/cardhost/app/templates/components/card-name-dialog.hbs

```diff
@@ -0,0 +1,16 @@
+<Dialog open={{true}} @title={{this.title}} @mode={{@mode}} @icon={{@icon}}>
+  <p class="dialog--message">
```

Comment:
All form element need labels. I think you could make this into the label and it would work well:

```html
<label class="dialog--message" for="card_name">
    Give your card a name:
</label>
<TextField id="card_name" />
```

Reply:
Good call, will do.